### ISSUE TITLE
google-visualization-errors with message "Cannot read property 'Cua' of undefined" when trying to use GeoChart version 2.0.29

### ISSUE THREAD

**Issue by samMutlak** [rakannimer/react-google-charts#376]
Our react application is using google GeoChart, it was working but now it always giving showing a google-visualization-error with message "Cannot read property 'Cua' of undefined".

I tried to find the  cause or find a workaround but did not succeeded until now... anybody encountered such problem? and any clue on what is the actual cause?
Thank you in advanced.

**Comment by samMutlak** [comment]
Actually it is working now, seems there was a change in one of the
dependencies of google-chart rebuilding & deploying the project again
solved the problem. but I could not locate the buggy library :(

**Comment by cgirbes** [comment]
Then, there is something that I'm not understanding, the problem in my Website persists. in addition the problem occurs eventually, not always.

If I am using the library in JS and HTML and referencing to the Google URL, there is no rebuilding & deploying needed. I'm lost in this issue :(

**Comment by samMutlak** [comment]
Yes, I am lost also, I tried to trace it but I could not... HW, since I did
not know how it was fixed, I do not know what to do if it appeared again.
waiting for someone else who can provide any useful details which we can
depend on.
